qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -264,6 +322,21 @@ def _set_prompt_active(self, val):
         self.setStyleSheet(style.get_stylesheet(self.STYLESHEET))
 
     @pyqtProperty(bool)
+    def command_active(self):
+        """Getter for self.command_active, so it can be used as Qt property."""
+        return self._command_active
+
+    def _set_command_active(self, val):
```

Comment:
This doesn't seem to be used anywhere - isn't it superseded by `set_mode_active`?